.gitignore

```diff
@@ -4,8 +4,10 @@ aster_weights/*
 experiments/*
 !experiments/.keep
 .idea/*
-__pycache__/*
+__pycache__
 projector/perceptual_weights/*
 !projector/perceptual_weights/lin/.keep
 !projector/perceptual_weights/vgg/.keep
-
+.pytest_cache
+.coverage
+.DS_Store
```

Comment:
end of file missing --> tu peux rajouter black à la CI? (c.f. la CI du commentaire plus haut)

Reply:
ca va je devrais m'en sortir sans snippet mdr